store/tikv/snapshot.go

```diff
@@ -462,7 +464,9 @@ func (s *tikvSnapshot) DelOption(opt kv.Option) {
 	case kv.ReplicaRead:
 		s.replicaRead = kv.ReplicaReadLeader
 	case kv.CollectRuntimeStats:
+		s.mu.Lock()
 		s.stats = nil
+		s.mu.Unlock()
```

Comment:
Almost LGTM for me, but please put the field `stats` into `mu` in the definition of `tikvSnapshot`.

Reply:
great, done